akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -185,6 +185,7 @@ object ActorMaterializer {
     system
   }
 
+  val DefaultMailbox: String = "akka.stream.materializer.default-mailbox"
```

Comment:
No reason to make this constant public API, move it into the internals/ActorMaterializerImpl somewhere